distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -209,6 +235,24 @@ get_atom_constructor atom = @Builtin_Method "Meta.get_atom_constructor"
 get_atom_fields : Atom -> Array
 get_atom_fields atom = @Builtin_Method "Meta.get_atom_fields"
 
+## PRIVATE
+
+   Get the constructors for the provided type.
+
+   Arguments:
+   - typ: The type to obtain the constructors for.
+get_type_constructors : Atom -> Array
```

Comment:
what is the type and documentation of `factory`?

Reply:
It has to be `Meta.Constructor.Value` - used here 1b89cadf069bbd5b7c433fa8ea62e7badb57874e

Reply:
Yes, but the type is not present in the signature of `get_type_constuctors` nor is in the list of arguments in the docs

Reply:
I see. Fixed in [74691a7](https://github.com/enso-org/enso/pull/5956/commits/74691a7bbb6db9eec5d3a8da9a750549db935c16)